lib/generators/loader-generator.js

```diff
@@ -49,9 +49,7 @@ const LoaderGenerator = webpackGenerator(
 		"examples/simple/src/lazy-module.js.tpl",
 		"examples/simple/src/static-esm-module.js.tpl"
 	],
-	[
-		"src/_index.js.tpl"
-	],
+	["src/_index.js.tpl"],
```

Comment:
no need

Reply:
I think `format` script / prettier the auto-formatting. Should I still restore to older formatting?

Reply:
y discard this

Reply:
I think CI would anyway run `npm run format` or `yarn format` and write the files so it's better to have it same in the source code to have consistency. :)